designer/apps/sitebranding/models.py

```diff
@@ -0,0 +1,92 @@
+""" Models related to the branding of individual sites """
+from wagtail.contrib.settings.models import BaseSetting, register_setting
+from wagtail.wagtailadmin.edit_handlers import FieldPanel, MultiFieldPanel
+from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+from django.db import models
+import re
+from django.core.exceptions import ValidationError
+
+
+def validate_hexadecimal_color(color):
+    """
+    Returns true if color is a string in the format hexadecimal format
+    ex: '#B62168', '#00a2e4'
+    Args:
+        color: (str) string representing hexadecimal color
+
+    Returns:
+        is_valid_hexadecimal_color: (bool) True if `color` is in valid hexadecimal format
+    """
+    if 7 != len(color):
+        raise ValidationError("Incorrect length. Hexadecimal colors must be 7 digits")
```

Comment:
Without the format in the error, this might be confusing. I think it'd be fine if both checks returned the same `"Incorrect format..."` error. You could delete this and either just `and` the conditions, or change your regex below to `r'^#[\dA-Fa-f]{6}$'` which would verify it the length as well.

Reply:
Sounds good!